lib/Paws/Net/Caller.pm

```diff
@@ -29,7 +29,7 @@ package Paws::Net::Caller;
         (defined $requestObj->content)?(content => $requestObj->content):(),
       }
     );
-    return ($response->{status}, $response->{content}, $response->{headers});
+    return $self->caller_to_response($service, $call_object, $response->{status}, $response->{content}, $response->{headers});
```

Comment:
send_request should not be calling caller_to_response anymore (that's part of the refactoring)